bundle/android/strings.xml

```diff
@@ -11,17 +11,14 @@
     <string name="gutenberg_native_add_block_before" tools:ignore="UnusedResources">Add Block Before</string>
     <string name="gutenberg_native_add_block_here" tools:ignore="UnusedResources">ADD BLOCK HERE</string>
     <string name="gutenberg_native_add_button_text" tools:ignore="UnusedResources">Add button text</string>
-    <string name="gutenberg_native_add_caption" tools:ignore="UnusedResources">Add caption</string>
```

Comment:
A quick way to verify that is safe to remove a localization key is by searching for this text in the code, if the text is referenced in a non-native file (basically a `*.js`) then it's ok to remove it as it will be included in the translations files.

Another way is to directly search for the text in the translation files (for example [`en-gb.json`](https://github.com/wordpress-mobile/gutenberg-mobile/blob/develop/src/i18n-cache/gutenberg/data/en-gb.json)), if it exists then it's safe to remove it.